cmd/juju/caas/add.go

```diff
@@ -84,6 +84,8 @@ Examples:
     juju add-k8s myk8scloud --controller mycontroller
     juju add-k8s --context-name mycontext myk8scloud
     juju add-k8s myk8scloud --region <cloudType/region>
+    juju add-k8s myk8scloud --cloud <cloudType>
```

Comment:
Should be --cloud-type
(to reinforce that it is not the cloud name)
Also, --cloud-type should just be for cloud not region. So the possibilities are:

--cloud-type ec2
--cloud-type ec2 --region us-east-1
--region ec2/us-east-1

Reply:
as discussed, the cloud could be name or type.
so we decide to use --cloud